pkg/crc/preflight/preflight.go

```diff
@@ -144,7 +144,8 @@ func doCleanUpPreflightChecks(checks []Check) error {
 func doRegisterSettings(cfg config.Schema, checks []Check) {
 	for _, check := range checks {
 		if check.configKeySuffix != "" {
-			cfg.AddSetting(check.getSkipConfigName(), false, config.ValidateBool, config.SuccessfullyApplied)
+			cfg.AddSetting(check.getSkipConfigName(), false, config.ValidateBool, config.SuccessfullyApplied,
+				"Skip preflight check (true/false, default: false)")
```

Comment:
This is different from all other booleans, which don't mention `true/false`, just the default.